File: bike_service_application/demo/src/main/java/com/aswin/demo/service/OwnerService.java
package com.aswin.demo.service;


import com.aswin.demo.entity.Bookings;
import com.aswin.demo.entity.Owner;
import org.springframework.stereotype.Service;

import java.util.List;

@Service
public interface OwnerService {
    public  String addowner(Owner owner);

    boolean isExist(String username, String password);

    List<Bookings> getAllBookings();
}

File: bikeservice_ui/src/service/OwnerService.js
import axios from "axios"
const API_URL = "http://localhost:8080"
class OwnerService{

    saveUser(owner){
        return axios.post(API_URL+"/ownwer/login",owner);
    }
    isExist(username, password){
        return axios.post(API_URL+"/owner/login/"+username+"/"+password,{})
    }
    getAllBookings(){
        return axios.get(API_URL+"/owner/allbooking")
    }

}

export default new OwnerService();